### ISSUE TITLE
Optimizing training procedure (learning rate, lr scheduling) for each architecture

### ISSUE THREAD

**Issue by zhan2977** [keras-team/autokeras#641]
### Feature Description

Currently, autokeras trains each architecture with the same learning rate and lr scheduling. Without hyperparamter tuning, the achieved performance is not optimal for each architecture. Besides architecture search, maybe we can also do learning rate search for each architecture.
